litex/soc/software/libcompiler_rt/Makefile

```diff
@@ -15,7 +15,7 @@ all: libcompiler_rt.a
 
 libcompiler_rt.a: $(OBJECTS)
 	$(CC) -c $(CFLAGS) $(1) $(SOC_DIRECTORY)/software/libcompiler_rt/mulsi3.c -o mulsi3.o
-	$(AR) crs libcompiler_rt.a $(OBJECTS)
+	$(AR) crs libcompiler_rt.a $(OBJECTS) mulsi3.o
```

Comment:
Why is this added?

Reply:
>mulsi3.c was compiled but never linked into libcompiler_rt.a, so I fixed that as well. The minimal variant will choke when linking the BIOS without this.

Meaning I guess no one ever actually checked that cores w/o multipliers worked in `litex`.

Reply:
Yes, and `libgcc` (or whatever spec `libgcc` follows) mandates it, but `compiler_rt` didn't have it until [February of this year](https://reviews.llvm.org/D42958), and only for RISC-V at that.